### ISSUE TITLE
Keyboard Navigation Doesn't Reach "Manage Content" 

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#3712]
### Description
When using keyboard navigation to go through each navigation element on the page, each section of the navigation bar is reached except for the items that are now nested under _Manage Content_: Manage Content, Manage Encoding Jobs, Manage Groups, Manage Users. These should be navigated through in sequence.

### Done Looks Like
- Keyboard navigation moves through the items listed under _Manage Content_.
   - These should come in sequence after the other items in the nav bar
